0-ad-history-mod-package/simulation/templates/gaia/tree/aleppo_pine.xml

```diff
@@ -3,7 +3,7 @@
   <Identity>
     <SpecificName>Aleppo Pine</SpecificName>
     <History>
-       The Aleppo Pine does not look like other pines. Instead of having the distinctive triangular shape of its relatives, the Aleppo’s canopy is rather like a dome. Aleppos’ height averages 65 ft. It is a hearty tree that can withstand dry and windy locations and can be found in the highlands of the Mediterranean.
+       The Aleppo Pine does not look like other pines. Instead of having the distinctive triangular shape of its relatives, the Aleppo’s canopy is rather like a dome. Aleppos’ height averages 65 feet(19.8 meters). It is a hearty tree that can withstand dry and windy locations and can be found in the highlands of the Mediterranean.
```

Comment:
```suggestion
       The Aleppo Pine does not look like other pines. Instead of having the distinctive triangular shape of its relatives, the Aleppo’s canopy is rather like a dome. Aleppos’ height averages 65 feet (19.8 meters). It is a hearty tree that can withstand dry and windy locations and can be found in the highlands of the Mediterranean.
```